Parameters.java

```diff
@@ -287,6 +291,12 @@ public static void load() {
 					alphabet = AlphabetType.AMINO_ACIDS.getValidCharacters();
 				}
 			}
+			if (map.get("outPath") != null){
+				outPath = (String) map.get("outPath");
+			}
+			if (map.get("outPrefix") != null){
+				outPrefix= (String) map.get("outPrefix");
+			}
```

Comment:
I don't think these if statements are necessary, since we aren't loading them in from `parameters.yml`. We should only keep this change if it makes sense to include "outPath" and "outPrefix" in `parameters.yml`.

Reply:
Ohh nice catch! Sorry -- I just noticed I didn't push a recent `parameters.yml` file here --- but in my own experiments, I do use both "outPath" and "outPrefix" in the yaml file -- I'll include those changes on my next push :)

Reply:
Ooh, sounds good!